profiles/takanome-dev.json

```
@@ -0,0 +1,6 @@
+{
+  "image": "https://camo.githubusercontent.com/2d8017444f17e2eb9cd2b304b3437de0efdc887543fb9f90f3e0942d608ef776/68747470733a2f2f7265732e636c6f7564696e6172792e636f6d2f646166777a736f64302f696d6167652f75706c6f61642f76313635313933343438382f6769746875622d73637265656e5f706e6763346b2e706e67",
+  "issueId": 1113,
+  "name": "Takanome Dev",
+  "username": "takanome-dev"
+}
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```